realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -517,21 +519,6 @@ public void or() {
         assertEquals(22, resultList.size());
     }
 
-    @Test(expected = UnsupportedOperationException.class)
```

Comment:
Some restriction lifted? In which case it should probably be noted in the CHANGELOG. Here and below.